lib/debug/repl.cpp

```diff
@@ -170,6 +178,30 @@ std::optional<std::unique_ptr<Repl>> Repl::fromFile(const std::string &path)
       repl->addUartCommands(bus.GetUint());
     }
   }
+
+  if (!sensors.HasMember("accelerometer")) {
+    logger_.log(core::LogLevel::kFatal,
+                "Missing required field 'sensors.accelerometer' in configuration file");
+    return std::nullopt;
+  }
+  const auto accelerometer = sensors["accelerometer"].GetObject();
+
```

Comment:
there's empty lines like this which we want to avoid here and in `addAccelerometerCommands`, remove them